Miscellaneous/docker/full_stack_example/config/beats/heartbeat/heartbeat.yml

```diff
@@ -0,0 +1,78 @@
+heartbeat.monitors:
+- type: icmp
+  schedule: '*/5 * * * * * *'
+  hosts: ["filebeat","metricbeat","packetbeat","elasticsearch","kibana","logstash","apache2","nginx","mysql"]
+  ipv4: true
+  ipv6: true
+  mode: any
+  timeout: 16s
+  wait: 1s
+- type: http
+  enabled: true
```

Comment:
I'd love to see some newlines (and comments) between sections. Every file in this repo is a "learning opportunity", so I think we should optimise for the _human_ reader as much as possible.